docs/UserGuide.md

```diff
@@ -1413,6 +1413,90 @@ Command Options: This command does not take in any arguments and will not proces
 
 #### 4.4.1 HelpCommand
 
+Get help for UniCa$h.
+
+Command: `help [<COMMAND_WORD>]`
+
+Command Argument: `<COMMAND_WORD>` is the command to get help for. If no 
+argument is specified, a general help message is shown as well as a pop up 
+containing a link to our User Guide.
+
+> To get a list of `<COMMAND_WORD>`, do `help` with no arguments
+
+##### Successful Execution
+
+###### Example 1
+
+> **Case**: Get general help.
+>
+> **Input**: `help`
+>
+> **Output**:
+> ```
+> Welcome to UniCa$h!
+>
+>For more detailed help on a command: help <command_word>
+>
+> Available Commands:
+> add_transaction
+> delete_transaction
+> edit_transaction
+> list
+> find
+> get
+> get_total_expenditure
+> summary
+> set_budget
```

Comment:
Seems like there's missing commands like `get_budget` and `clear_budget`? Not sure how we're progressing with these

Reply:
Got it